src/app/pages/account/users/user-list-details/user-list-details.component.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  Component, OnInit, ChangeDetectionStrategy, Input, EventEmitter, Output, ViewChild,
+  OnDestroy, ChangeDetectorRef,
+} from '@angular/core';
+import { MatRowDef, MatTableDataSource, MatColumnDef } from '@angular/material/table';
+import { untilDestroyed, UntilDestroy } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import _ from 'lodash';
+import { Group } from 'app/interfaces/group.interface';
+import { Option } from 'app/interfaces/option.interface';
+import { User } from 'app/interfaces/user.interface';
+import { UserFormComponent } from 'app/pages/account/users/user-form/user-form.component';
+import { DialogFormConfiguration } from 'app/pages/common/entity/entity-dialog/dialog-form-configuration.interface';
+import { EntityDialogComponent } from 'app/pages/common/entity/entity-dialog/entity-dialog.component';
+import { EntityUtils } from 'app/pages/common/entity/utils';
+import { IxTableComponent } from 'app/pages/common/ix-tables/components/ix-table/ix-table.component';
+import { ModalService } from 'app/services';
+import { DialogService } from 'app/services/dialog.service';
+import { WebSocketService } from 'app/services/ws.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-user-list-details',
+  templateUrl: './user-list-details.component.html',
+  styleUrls: ['./user-list-details.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class UserListDetailsComponent implements OnInit, OnDestroy {
+  @Input() expandedRow: User;
+  @Input() dataSource: MatTableDataSource<User>;
+  @Input() colspan: number;
+  @Input() users: [User[]?] = [];
+  @Output() update = new EventEmitter<unknown>();
+  @ViewChild(MatRowDef, { static: false }) rowDef: MatRowDef<User>;
+  @ViewChild(MatColumnDef, { static: false }) columnDef: MatColumnDef;
+
+  groups: [Group[]?] = [];
```

Comment:
Looks like we can save on a request by only querying for a group when user is about to be deleted.